src/qz/printer/action/PrintHTML.java

```diff
@@ -192,7 +192,7 @@ public void print(PrintOutput output, PrintOptions options) throws PrinterExcept
             settings.setJobName(pxlOpts.getJobName(Constants.HTML_PRINT));
             settings.setPrintQuality(PrintQuality.HIGH);
 
-            if (pxlOpts.getColorType() != null) {
+            if (pxlOpts.getColorType().getAsChromaticity() != null) {
```

Comment:
Why wouldn't this be `pxlOpts.getColorType() != DEFAULT`?  That is more readable in my opinion.  It also should have a comment explaining that it will fallback to the default value of the printer, since this behavior doesn't seem to be documented anywhere (Implied by name, but never explicitly stated).